src/node.cc

```diff
@@ -1510,15 +1511,16 @@ void AppendExceptionLine(Environment* env,
 
   Local<String> arrow_str = String::NewFromUtf8(env->isolate(), arrow);
 
-  if (!arrow_str.IsEmpty() && !err_obj.IsEmpty() && err_obj->IsNativeError()) {
+  if (!arrow_str.IsEmpty() && !err_obj.IsEmpty() &&
+      (!handlingFatalError || err_obj->IsNativeError())) {
```

Comment:
@bnoordhuis Definitely not disagreeing with that. :) I know you’re not a fan of comments when the code can be self-explanatory, but I’ve still added a commit that inverts the logic + a comment that goes through the various scenarios. I think I would have found that helpful when looking at the code, but if you don’t like it or have different suggestions, I’m happy to hear that.